models/enterprise_sonic/pim_global/merged_example_01.txt

```diff
@@ -0,0 +1,33 @@
+# Using merged
+#
+# Before State:
+# -------------
+#
+# sonic# show running-configuration | grep "ip pim"
+# ip pim vrf default ssm prefix-list prefix-list-1
+# sonic#
+
```

Comment:
It would be worthwhile to include an example here for changing an existing value for some attribute to a new value (e.g. ssm prefix-list or join_prune_interval).

Reply:
Have updated the example to include modification of existing configuration.